qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/presenters/CreateOfferPresenter.groovy

```diff
@@ -50,7 +57,29 @@ class CreateOfferPresenter implements CreateOfferOutput, ListProductsOutput{
 
     @Override
     void showAvailableProducts(List<Product> availableProducts) {
-        this.createOfferViewModel.foundProducts.clear()
-        this.createOfferViewModel.foundProducts.addAll(availableProducts)
+        this.createOfferViewModel.sequencingProducts.clear()
+        this.createOfferViewModel.managementProducts.clear()
+        this.createOfferViewModel.primaryAnalysisProducts.clear()
+        this.createOfferViewModel.secondaryAnalysisProducts.clear()
+        this.createOfferViewModel.storageProducts.clear()
```

Comment:
should be done by the `Presenter`

Reply:
Why not in the CreateOfferPresenter?

Reply:
yes, that is what I meant. Just any presenter that does update those values should take care of removing the old values if this should be done

Reply:
This is outdated now. I removed this part since I want to keep products that have been in the list before

Reply:
If you have further rejections let me know